src/themes/default/components/core/blocks/Reviews/Reviews.vue

```diff
@@ -119,10 +119,17 @@ export default {
       }
     }
   },
-  computed: {
-    product () {
-      return this.$store.state.product
+  props: {
+    originalProductId: {
```

Comment:
Only this one is being used so I believe we should refactor the interface to have single “productId”

Reply:
As i wrote to you already in #developement i used the originalId for submitting first, but the review didn't show up in the M2.2 Backend, while using ProductId (Configurable Children) it worked fine. until this one: https://github.com/DivanteLtd/vue-storefront/blob/bd559f1baad7cd392bc5bae7b935a60484e2e6e5/src/store/modules/product.js#L323 is solved, i thought it's better to pass both.

correct me, if i am wrong

Reply:
This is not that case this check is required for a case when somebody is trying to get to the single product page assigned to configurable one. It’s redirecting to the configurable product in that caseX 

We need to modify the way reviews are retrieved to retrieve all reviews related to all configurable children variants not only to the parent product